redis/internal/redisprom/pools.go

```diff
@@ -3,12 +3,13 @@ package redisprom
 import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/reddit/baseplate.go/internalv2compat"
```

Comment:
you have to add `//lint:ignore SA1019` before to silence staticcheck. example: https://github.com/reddit/baseplate.go/blob/9814cae3aaf1c265c18461680590a766e390d5a6/kafkabp/prometheus.go#L6